cxx-qt-gen/test_outputs/custom_default.rs

```diff
@@ -34,22 +35,23 @@ mod my_object {
         type QVariant = cxx_qt_lib::QVariantCpp;
 
         #[rust_name = "public"]
-        fn getPublic(self: &MyObject) -> i32;
+        fn getPublic(self: &MyObjectQt) -> i32;
         #[rust_name = "set_public"]
-        fn setPublic(self: Pin<&mut MyObject>, value: i32);
+        fn setPublic(self: Pin<&mut MyObjectQt>, value: i32);
 
         #[rust_name = "new_cpp_object"]
-        fn newCppObject() -> UniquePtr<MyObject>;
+        fn newCppObject() -> UniquePtr<MyObjectQt>;
     }
 
     extern "Rust" {
+        #[cxx_name = "MyObjectRust"]
         type RustObj;
```

Comment:
this will in future be the real `MyObject`, right?

But I guess that's out of scope for now :thinking:

Reply:
Right, one of the next changes is to drop the `Data` struct and just have the `RustObj` struct. Then add the qobject macro and use the RustObj struct ident as the QObject ident.... baby steps :-)

Reply:
(and potentially either within those changes or separately moving the properties from C++ -> Rust)